examples/Microsoft.Network_virtualNetworks@2024-05-01/with_ipam_pool/main.tf

```diff
@@ -0,0 +1,126 @@
+terraform {
+  required_providers {
+    azapi = {
+      source = "Azure/azapi"
+    }
+    azurerm = {
+      source = "Hashicorp/azurerm"
+    }
+    time = {
+      source = "Hashicorp/time"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
+
+provider "azapi" {
+  skip_provider_registration = false
+}
+
+variable "resource_name" {
+  type    = string
+  default = "acctest0001"
+}
+
+variable "location" {
+  type    = string
+  default = "westeurope"
+}
+
+resource "azapi_resource" "resourceGroup" {
+  type     = "Microsoft.Resources/resourceGroups@2020-06-01"
+  name     = var.resource_name
+  location = var.location
+}
+
+data "azurerm_client_config" "current" {
+}
+
+data "azapi_resource" "subscription" {
+  type                   = "Microsoft.Resources/subscriptions@2021-01-01"
+  resource_id            = "/subscriptions/${data.azurerm_client_config.current.subscription_id}"
+  response_export_values = ["*"]
+}
+
+resource "azapi_resource" "networkManager" {
+  type      = "Microsoft.Network/networkManagers@2024-05-01"
+  parent_id = azapi_resource.resourceGroup.id
+  name      = var.resource_name
+  location  = var.location
+  body = {
+    properties = {
+      description                 = ""
+      networkManagerScopeAccesses = []
+      networkManagerScopes = {
+        managementGroups = []
+        subscriptions = [
+          data.azapi_resource.subscription.id,
+        ]
+      }
+    }
+    tags = { "sampleTag" : "sampleTag" }
```

Comment:
The `azapi_resource` also supports `tags` in the root level, which is similar with `azurerm`, please move it to the root level.

Reply:
fixed